plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend

```diff
@@ -253,11 +253,15 @@ class TextualScenarioValidator extends AbstractTextualScenarioValidator {
 	@Check
 	def checkIfACombinedFragmentIsUsedAfterDeleteMessage(CombinedFragment combinedFragment) {
 		var model = TextualScenarioHelper.getModelContainer(combinedFragment)
-		var elements = (model as Model).elements
+		checkCombinedFragmentAfterDelete(model as Model, combinedFragment)
+	}
+	

```

Comment:
Name too long checkIfACombinedFragmentIsUsedAfterDeleteMessage.
Can be removed and keep only checkCombinedFragmentAfterDelete.